src/Domain/Entities/Member.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Codidact.Domain.Common;
+using System;
+
+namespace Codidact.Domain.Entities
+{
+    /// <summary>
+    /// An Entity that represents a member of the community
+    /// </summary>
+    public class Member : AuditableEntity
+    {
+        /// <summary>
+        /// Auto Incremented Identification number
+        /// </summary>
+        public long Id { get; set; }
+        
+        /// <summary>
+        /// The display name that is to be displayed for the member
+        /// </summary>
+        public string DisplayName { get; set; }
+
+        /// <summary>
+        /// The bio information of the member
+        /// </summary>
+        public string Bio { get; set; }
+
+        /// <summary>
+        /// The main email of the member
+        /// </summary>
+        public string Email { get; set; }
+
+        /// <summary>
+        /// Geographical location of the member
+        /// </summary>
+        public string Location { get; set; }
+
+        /// <summary>
+        /// Whether the member is imported from StackExchange or not
+        /// </summary>
+        public bool IsFromStackExchange { get; set; }
+
+        /// <summary>
+        /// The StackExchange Id if it was imported
+        /// </summary>
+        public long? StackExchangeId { get; set; }
+
+        /// <summary>
+        /// When was the data validated against StackExchange
+        /// </summary>
+        public DateTime? StackExchangeValidated { get; set; }
+
+        /// <summary>
+        /// When was the last time the data was imported from StackExchange
+        /// </summary>
+        public DateTime? StackExchangeLastImported { get; set; }
+
+        /// <summary>
+        /// Whether the email has been verified for the member
+        /// </summary>
+        public bool IsEmailVerified { get; set; }
+
+        /// <summary>
+        /// Whether the member is suspended
+        /// </summary>
+        public bool IsSuspended { get; set; }
+
+        /// <summary>
+        /// When does the suspension end
+        /// </summary>
+        public DateTime? SuspensionEndDate { get; set; }
```

Comment:
Do we intend to keep track of past suspensions?

Reply:
From a moderation perspective, that's kind of necessary. Being able to see at a glance, natively, the history of a user is fairly important.

Reply:
We'll need to add a UserHistory table as we go as well. We should still retain this field, though, because it makes lookup and calculation easy.